src/App.js

```diff
@@ -1,79 +1,162 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
 import "./App.css";
-import picture from "./mood-smiley-face.jpg";
+import picture1 from "./nidhi.jpg";
+import picture2 from "./Nidhi_Gretch.jpg";
+import picture3 from "./Nidhi_Farha.jpg";
+
+import thumbnail1 from "./nidhi.jpg";
+import thumbnail2 from "./Nidhi_Gretch.jpg";
+import thumbnail3 from "./Nidhi_Farha.jpg";
+
+import ImageGallery from "react-image-gallery";
+
 import {
   Row,
   Col,
   Image,
   Button,
   ButtonToolbar,
-  FormControl
+  FormControl,
+  FormGroup,
+  ControlLabel,
+  HelpBlock
 } from "react-bootstrap";
-import { ProfileImage } from "./component";
+
+import { CenterLines } from "./CenterLines";
+import { CreateTable } from "./ContactForm";
+
+import { FieldGroup } from "./ContactForm";
+
+import { FormButton } from "./ContactForm";
+
+import { ProfileImage, WrapperDiv, SlideShow } from "./component";
+
+import "./style.css";
 
 class App extends Component {
   constructor(props) {
-    super();
+    super(props);
     this.state = {
-      text: "WOW"
+      name: "",
+      email: "",
+      message: ""
     };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    console.log(this.state);
+    const target = e.target;
+    const value = target.value;
+    const id = target.id;
+
+    this.setState({
+      [id]: value
+    });
+  }
+
+  async handleSubmit() {
+    // alert('name: ' + this.state.name + ' ' + 'email: ' + this.state.email + ' ' + 'message: ' + this.state.message);
+    try {
+      let res = await fetch("http://localhost:8080/contact", {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json"
+        }),
+        body: JSON.stringify(this.state)
+      });
+
+      res = await res.json();
+      console.log(res);
+
+      var y = res.value;
+      console.log(y);
+    } catch (err) {
+      console.error(err);
+    }
+
+    if (y === 1) {
```

Comment:
glad to see you started dabbling with returning different responses from the api! typically we wouldnt set values such as 1, 2, etc. but i still love the thought process!